src/pipx/commands/common.py

```diff
@@ -174,11 +176,12 @@ def get_package_summary(
         python_version,
         python_path,
         package_metadata.package_version,
-        package,
+        package_metadata.package or package,
```

Comment:
I think this should not be necessary if you get `package` correct in the first place (see above comment.)